includes/configuration_managers/class-newspack-gam-configuration-manager.php

```diff
@@ -0,0 +1,113 @@
+<?php
+/**
+ * Site Kit plugin configuration manager.
+ *
+ * @package Newspack
+ */
+
+namespace Newspack;
+
+defined( 'ABSPATH' ) || exit;
+
+require_once NEWSPACK_ABSPATH . '/includes/configuration_managers/class-configuration-manager.php';
+
+/**
+ * Provide an interface for configuring and querying the configuration of Google Site Kit.
+ */
+class Newspack_GAM_Configuration_Manager extends Configuration_Manager {
+
+	/**
+	 * The slug of the plugin.
+	 *
+	 * @var string
+	 */
+	public $slug = 'newspack-gam';
+
+	/**
+	 * Get whether the Site Kit plugin is active and set up.
+	 *
+	 * @return bool Whether Site Kit is active and set up.
+	 */
+	public function is_configured() {
+		return class_exists( 'Newspack_GAM_Model' );
+	}
+
+	/**
+	 * Get the ad units from our saved option.
+	 */
+	public function get_ad_units() {
+		return $this->is_configured() ?
+			\Newspack_GAM_Model::get_ad_units() :
+			$this->unconfigured_error();
+	}
+
+	/**
+	 * Get a single ad unit.
+	 *
+	 * @param number $id The id of the ad unit to retrieve.
+	 */
```

Comment:
Minor: It'd be good to have the `@return` documented in these methods too.

Reply:
Yup, good call: https://github.com/Automattic/newspack-plugin/pull/184/commits/07062063fe08d9ef3779d158e91af3233447f9a9